src/users/usersActions.js

```diff
@@ -20,14 +17,7 @@ export const reportPresence = (hasFocus: boolean = true, newUserInput: boolean =
     return; // not logged in
   }
 
-  const now = new Date();
-  if (differenceInSeconds(now, lastReportPresence) < 60) {
-    // TODO throttle properly; probably fold setInterval logic in here
```

Comment:
Commit-level comments on this commit:
e12c452df presence: strip throttling from `reportPresence` wrapper

Nit: "would fix #3699 as literally reported" is no longer quite right, as we've edited #3699 so it describes the problem more broadly.

Related nit: "would fix #..." is a dangerous phrase to put in a commit message, because GitHub sees the "fix" and doesn't understand the "would". :-) Should basically always get reworded to hide from GitHub's auto-close rules, with e.g. a circumlocution like "would be a fix for #...".

Stepping back, though, I think my comment on this commit from the previous PR still applies:
https://github.com/zulip/zulip-mobile/pull/3704#pullrequestreview-325863916
> This looks like a good implementation of what it says on the tin. I think it may not be part of what we want following today's discussion, though.

Reply:
> Stepping back, though, I think my comment on this commit from the previous PR still applies:
> [#3704 (review)](https://github.com/zulip/zulip-mobile/pull/3704#pullrequestreview-325863916)
> 
> > This looks like a good implementation of what it says on the tin. I think it may not be part of what we want following today's discussion, though.

The fix for this is basically to squash the second commit into the fourth commit. As there's not much substance to it, done. (Although this does make the commit message uncomfortably crowded. :confused:)

Reply:
I like the result, thanks!

The commit message does get a bit longer. I think it might actually on net be simpler than before, though. One way it's certainly simpler is that the "If the user logs in and out" bullet previously had to tell a multi-step story ("This was previously concealed by ... which was itself removed in a previous patch") and that's now a simple before/after.